modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -130,6 +130,14 @@ using GRunArgP = util::variant<
     >;
 using GRunArgsP = std::vector<GRunArgP>;
 
+inline std::vector<GRunArgP>& operator += (std::vector<GRunArgP> &lhs, const std::vector<GRunArgP> &rhs);
```

Comment:
Why don't you use the alias GRunArgsP instead of std::vector<GRunArgP >?

Reply:
Replaced `std::vector<GRunArgP>` with `GRunArgsP`